wasi-sdk.cmake

```diff
@@ -0,0 +1,34 @@
+# Cmake toolchain description file for the Makefile
+
+# This is arbitrary, AFAIK, for now.
+cmake_minimum_required(VERSION 3.4.0)
```

Comment:
Hmm.. we should probably consider including this in the wasi-sdk itself, or at least the template version (WASI_SDK_PREFIX would need to be injected somehow).

Reply:
I don't love that this is top level file.. but I can't think of a better place for it.   Could you add a comment saying `TODO: get this added to the wasi-sdk itself so we don't need to duplicate it here`?

Reply:
That's a good idea. We can try to add this on the wasi-sdk, and once is added remove it from here :)